tests/input/cpp/bug_1139.cpp

```diff
@@ -0,0 +1,31 @@
+void a()
+{
+               if((tmp == nullptr) ||
+                        ((tmp->type != CT_NUMBER) &&
+                         (tmp->type != CT_SIZEOF) &&
+                         !(tmp->flags & (PCF_IN_STRUCT | PCF_IN_CLASS))) ||
+                        (tmp->type == CT_NEWLINE)
+                        )
+               {
+                  set_chunk_type(next, CT_LABEL_COLON);
+               }
+else if ((tmp == nullptr) ||
+                        ((tmp->type != CT_NUMBER) &&
+                         (tmp->type != CT_SIZEOF) &&
+                         !(tmp->flags & (PCF_IN_STRUCT | PCF_IN_CLASS))) ||
+                        (tmp->type == CT_NEWLINE)
+                        )
```

Comment:
You are not testing if force works correctly. Pleas make one branch with 
```cpp
... (tmp->type == CT_NEWLINE))
```
and the other with
```cpp
... (tmp->type == CT_NEWLINE)

)
```